src/openms/include/OpenMS/ANALYSIS/XLMS/OPXLDataStructs.h

```diff
@@ -184,7 +184,7 @@ namespace OpenMS
        */
       struct CLSMScoreComparator
       {
-        bool operator() (const CrossLinkSpectrumMatch& a, const CrossLinkSpectrumMatch& b)
+        inline bool operator() (const CrossLinkSpectrumMatch& a, const CrossLinkSpectrumMatch& b)
```

Comment:
no need to specify `inline` here, since you provide the implementation within the class declaration (implicit inline)

Reply:
see https://isocpp.org/wiki/faq/inline-functions#inline-member-fns-more

Reply:
does it hurt to make the intent explicit? in case these functions end up in the cpp file after future refactoring.

Reply:
https://stackoverflow.com/questions/5808498/function-inlining-what-are-examples-where-it-hurt-performance

Reply:
I removed the redundant inlines and some from functions that have more than two lines of code. Did not affect the runtime of my benchmark test, so it should be fine this way.